server/plugin/plugin.go

```diff
@@ -64,6 +65,7 @@ func (p *Plugin) OnActivate() error {
 	p.http = myhttp.NewService(mux.NewRouter(), p.apps,
 		dialog.Init,
 		helloapp.Init,
+		api.Init,
```

Comment:
`restapi.Init,` would be more intention-revealing

Reply:
Done